src/cli.ts

```diff
@@ -1,51 +1,537 @@
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('min-height', {
+      defaultDescription: '0',
+      description: 'set window minimum height in pixels',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('min-width', {
+      defaultDescription: '0',
+      description: 'set window minimum width in pixels',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('process-envs', {
+      coerce: getProcessEnvs,
+      description:
+        'a JSON string of key/value pairs to be set as environment variables before any browser windows are opened',
+      group: 'App Window Options',
+      type: 'string',
+    })
+    .option('single-instance', {
+      default: false,
+      description: 'allow only a single instance of the application',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('tray', {
+      default: false,
+      description:
+        "Allow app to stay in system tray. If 'start-in-tray' is set as argument, don't show main window on first start",
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('width', {
+      defaultDescription: '1280',
+      description: 'app window default width in pixels',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('x', {
+      description: 'set window x location in pixels from left',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('y', {
+      description: 'set window y location in pixels from top',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('zoom', {
+      default: 1.0,
+      description: 'default zoom factor to use when the app is opened',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    // Internal Browser Options
+    .option('file-download-options', {
+      coerce: parseJson,
+      description:
+        'a JSON string of key/value pairs to be set as file download options. See https://github.com/sindresorhus/electron-dl for available options.',
+      group: 'Internal Browser Options',
+      type: 'string',
+    })
+    .option('honest', {
+      default: false,
+      description:
+        'prevent the normal changing of the user agent string to appear as a regular Chrome browser',
+      group: 'Internal Browser Options',
+      type: 'boolean',
+    })
+    .option('inject', {
+      description:
+        'path to a CSS/JS file to be injected. Pass multiple times to inject multiple files.',
+      group: 'Internal Browser Options',
+      type: 'array',
+    })
+    .option('lang', {
+      defaultDescription: 'os language at runtime of the app',
+      description:
+        'set the language or locale to render the web site as (e.g., "fr", "en-US", "es", etc.)',
+      group: 'Internal Browser Options',
+      type: 'string',
+    })
+    .option('u', {
+      alias: 'user-agent',
```

Comment:
I was seeing

```
Internal Browser Options
      --file-download-options  a JSON string of key/value pairs to be set as file download options. See
                               https://github.com/sindresorhus/electron-dl for available options.       [string]
      --honest                 prevent the normal changing of the user agent string to appear as a regular
                               Chrome browser                                         [boolean] [default: false]
      --inject                 path to a CSS/JS file to be injected. Pass multiple times to inject multiple
                               files.                                                                    [array]
      --lang                   set the language or locale to render the web site as (e.g., "fr", "en-US", "es",
                               etc.)                       [string] [default: os language at runtime of the app]
  -u, --user-agent             set the app user agent string                                            [string]
```

And it made me think: would yargs support aliasing options? It'd be nice to alias `--user-agent-honest` with `--honest` (with `--user-agent-honest` being the "main" one). Goals:

1. Proximity of `--user-agent` and `--user-agent-honest` (right now, because of alphabetical sorting --which is good--, there are three options between them, hindering discoverability 😕)
2. Backwards compatibility (we want to keep honoring `--honest` for users using it, but we want to direct new users to `--user-agent-honest`

Reply:
> I was seeing
> 
> ```
> Internal Browser Options
>       --file-download-options  a JSON string of key/value pairs to be set as file download options. See
>                                https://github.com/sindresorhus/electron-dl for available options.       [string]
>       --honest                 prevent the normal changing of the user agent string to appear as a regular
>                                Chrome browser                                         [boolean] [default: false]
>       --inject                 path to a CSS/JS file to be injected. Pass multiple times to inject multiple
>                                files.                                                                    [array]
>       --lang                   set the language or locale to render the web site as (e.g., "fr", "en-US", "es",
>                                etc.)                       [string] [default: os language at runtime of the app]
>   -u, --user-agent             set the app user agent string                                            [string]
> ```
> 
> And it made me think: would yargs support aliasing options? It'd be nice to alias `--user-agent-honest` with `--honest` (with `--user-agent-honest` being the "main" one). Goals:
> 
> 1. Proximity of `--user-agent` and `--user-agent-honest` (right now, because of alphabetical sorting --which is good--, there are three options between them, hindering discoverability 😕)
> 2. Backwards compatibility (we want to keep honoring `--honest` for users using it, but we want to direct new users to `--user-agent-honest`

So like this?
```
Internal Browser Options
      --file-download-options        a JSON string of key/value pairs to be set as file download options. See https://github.com/sindresorhus/electron-dl for available options.
                                                                                                                                                                          [string]
      --inject                       path to a CSS/JS file to be injected. Pass multiple times to inject multiple files.                                                   [array]
      --lang                         set the language or locale to render the web site as (e.g., "fr", "en-US", "es", etc.)  [string] [default: os language at runtime of the app]
  -u, --user-agent                   set the app user agent string                                                                                                        [string]
      --user-agent-honest, --honest  prevent the normal changing of the user agent string to appear as a regular Chrome browser                         [boolean] [default: false]
```

Reply:
> So like this?

I had in mind not keeping `--honest` (but still honoring it), but if what you show is what's easy with `yargs`, perfect.

Reply:
We could do it with a bit of extra code (make a separate option for `--honest` that is hidden, and when set, manually pass it on to `--user-agent-honest`). Up to you.

Reply:
> We could do it with a bit of extra code (make a separate option for `--honest` that is hidden, and when set, manually pass it on to `--user-agent-honest`). Up to you.

@TheCleric nah, let's not go here. What you suggested is great and actually better to limit confusion for existing users.